_tuning-your-cluster/index.md

```diff
@@ -192,11 +192,25 @@ To better understand and monitor your cluster, use the [CAT API]({{site.url}}{{s
 
 ## (Advanced) Step 6: Configure shard allocation awareness or forced awareness
 
+To further fine-tune your shard allocation, you can set custom node attributes for shard allocation awareness or forced awareness.
+
 ### Shard allocation awareness
 
-If your nodes are spread across several geographical zones, you can configure shard allocation awareness to allocate all replica shards to a zone that’s different from their primary shard.
+You can set custom node attributes on OpenSearch nodes to be used for shard allocation awareness. For example, you can set the `zone` attribute on each node to represent the zone in which the node is located. You can also use the `zone` attribute to ensure that primary shard and its replica shards are allocated in a balanced manner across available distinct zones, for example, maximum shard copies per zone would equal `ceil (number_of_shard_copies/number_of_distinct_zones)`.
+
+Shard allocation awareness attempts to separate primary and replica shards across multiple zones, because two shard copies can not be placed on the same node. When only one zone is available, such as after a zone failure, OpenSearch allocates replica shards to the only remaining zone. For example, if your index has a total 5 shard copies (1 primary and 4 replicas) and nodes in 3 distinct zones, then OpenSearch will perform the following to allocate all 5 shard copies: 
```

Comment:
```suggestion
Shard allocation awareness attempts to separate primary and replica shards across multiple zones because 2 shard copies cannot be placed on the same node. When only 1 zone is available, such as after a zone failure, OpenSearch allocates replica shards to the only remaining zone. For example, if your index has a total of 5 shard copies (1 primary and 4 replicas) and nodes in 3 distinct zones, then OpenSearch will perform the following to allocate all 5 shard copies: 
```